src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/ReadyToRunHelperNode.cs

```diff
@@ -22,11 +22,11 @@ public enum ReadyToRunHelperId
         GetThreadStaticBase,
         DelegateCtor,
         ResolveVirtualFunction,
-
         TypeHandle,
         FieldHandle,
         MethodDictionary,
         MethodEntry,
+        ResolveGenericVirtualMethod
```

Comment:
No we don't need one, but I thought it was nicer than re-using another since they're named specifically and I didn't want to go down the VirtualCall path and have to special-case GVMs.  Fadi will probably add on his GVM work here, too.